Kitodo/src/main/resources/kitodo_config.properties

```diff
@@ -465,8 +465,8 @@ exportWithoutTimeLimit=true
 # LDAP Configuration
 # -----------------------------------
 
-# Logins ueber LDAP verwenden
-ldap_use=false
+# Use logins via LDAP
+ldap_use=true
```

Comment:
Why are you enabling LDAP authentication by default?

Reply:
See [here](https://github.com/kitodo/kitodo-production/issues/456#issuecomment-723027253):

> Procedure proposal:
> 
>     1. Enable individual users are authenticated against database even when LDAP is on. (#4014)
> 
>     2. Deactivate the LDAP group for the users supplied with the basic database. (Cf. also #3304)
> 
>     3. Remove the LDAP on/off switch. (#3336)
>        (The ldap_use parameter in the kitodo_config.properties file should initially be retained
>        so that users can initially remove their LDAP group after a migration. However, the
>        default for this attribute can be set to true for new installations.)

Reply:
Than add this as explanation. Without this I would deactivate this again after this pull request is merged!

I hope that your migration way of LDAP usage will be documented somewhere in an official way as issues and pull requests are not a long term documentation.

Reply:
As I wrote: You would want to leave the setting unchanged after a migration. It is only about empty new installations, where this can be `true` in the future (does not have to be. Anyone can set it to `false` by hand. If someone did that, they should be aware that LDAP does not work.)